### DISCUSSION TITLE
Add Password save on Jellyfin for Android TV

### DISCUSSION THREAD

**Discussion by LoMGolden** [jellyfin/jellyfin-androidtv#758] [General]
I think it might be interesting to have an option to save the password locally on the Android TV. Because users are regularly disconnected (every 3 weeks / 1 month) from Android TV and are forced to re-enter their password with the remote control (not practical at all). This problem does not appear on Jellyfin applications of other media.

**Comment by nielsvanvelzen** [comment]
We're already working on this. Our priority for the next release is to create a brand new authentication flow. This includes adding multiple servers and remembering multiple users. You can find more information in #612.
